include/net/nrf_cloud.h

```diff
@@ -204,6 +226,29 @@ struct nrf_cloud_tx_data {
 	enum mqtt_qos qos;
 };
 
+/** @brief Cell-based location request type */
+enum cell_based_location_type {
+	CELL_LOC_TYPE_INVALID = -1,
+	CELL_LOC_TYPE_SINGLE,
+	CELL_LOC_TYPE_MULTI /* Not yet supported */
+};
+
+/**@brief Structure used for cell based location results from nRF Cloud. */
+struct cell_based_loc_data {
+	enum cell_based_location_type type;
+	double lat;
+	double lon;
+	uint32_t unc;
+};
+
+struct nrf_cloud_pgps_result {
```

Comment:
needs doxygen

Reply:
this can live here if you want to use it in yours pgps stuff 😄 
Otherwise, yes it will be only in rest.h